x-pack/plugins/fleet/public/applications/integrations/sections/epm/screens/home/available_packages.tsx

```diff
@@ -132,8 +137,12 @@ function getAllCategoriesFromIntegrations(pkg: PackageListItem) {
     return pkg.categories;
   }
 
-  const allCategories = pkg.policy_templates?.reduce((accumulator, integration) => {
-    return [...accumulator, ...(integration.categories || [])];
+  const allCategories = pkg.policy_templates?.reduce((accumulator, policyTemplate) => {
+    if (isInputOnlyPolicyTemplate(policyTemplate)) {
+      // input only policy templates do not have categories
```

Comment:
Is there reason for concern here for when we go to browse/search input packages in the integrations grid UI? Wondering if this is just a missing field from the sample package we're using or an intentional omission from the package spec.

Reply:
Input packages can't have integrations (for now), so they only have the top level categories as opposed to policy template categories.